src/components/Paginator.jsx

```diff
@@ -0,0 +1,46 @@
+import React from "react"
+
+class Paginator extends React.Component {
```

Comment:
constructor is missing:
https://www.digitalocean.com/community/tutorials/react-constructors-with-react-components

Reply:
> The constructor comes in very handy, but sometimes it’s just too much. If you aren’t planning on maintaining state in your component and aren’t binding any event handlers, you don’t need to define a constructor at all.

So I agree, this one should have a constructor for binding the event handlers 👍